tests/voq/test_voq_nbr.py

```diff
@@ -436,12 +450,25 @@ def _change_vm_interface_on_vm(nbrhosts, state="up", node=None, results=None):
         for eos_intf in nbr['conf']['interfaces'].keys():
             if "Loopback" in eos_intf:
                 continue
+
             if state == "up":
-                logger.info("Startup EOS %s interface %s", node, eos_intf)
-                node_results.append(nbr['host'].eos_config(lines=["no shutdown"], parents=["interface %s" % eos_intf]))
+                logger.info("Startup Nbr %s interface %s", node, eos_intf)
```

Comment:
can we use the Fanout class functions to shutdown and startup interface iso doing if/else here
https://github.com/sonic-net/sonic-mgmt/blob/master/tests/common/devices/fanout.py has these function defined for different type of fanout devices

Reply:
The function 'change_vm_interfaces' is used in tests test_neighbor_clear_one and test_neighbor_clear_all. The intent of these test is to clear the neighbor without bringing the actual port on the linecard down. So, bringing the fanout port down, changes the intent of the test entirely.

Reply:
@arlakshm can you help review test_voq_nbr changes